### DISCUSSION TITLE
Touch-on-change configuration for automatically discovered fields

### DISCUSSION THREAD

**Discussion by pzerelles** [pablo-abc/felte#131] [Q&A]
Hi, is there a way to set `touchOnChange` to `true` for regular text input fields. For forms that are editors and have already values, I would prefer to show input errors after changes to a field immediately, not only after focus out.

**Comment by pablo-abc** [comment]
There's no configuration option to allow this behaviour as of now. But, the only thing that defines whether validation messages are shown is if the field is "touched". So if you set your field to touched as soon as you start typing (e.g. by using the `setTouched` helper), validation messages will show immediately!
